app/pages/[page].js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { applySession } from "next-session";
+import { useRouter } from "next/router";
+import { Forms, getHandler } from "../form-schema";
+import Button from "@button-inc/bcgov-theme/Button";
+
+export default function home({ formIndex, formData, validPage, prevPageUrl }) {
+  const Form = Forms[formIndex];
+  const router = useRouter();
+  const onFirstPage = prevPageUrl === -1;
+
+  const rerouteHandler = (nextPage, _isValid, lastPage) => {
+    router.push(lastPage ? "/end" : nextPage);
+  };
+
+  const handleBackClick = () => {
+    if (onFirstPage) return;
+    router.push(prevPageUrl);
+  };
```

Comment:
Some boilerplate for handling backward navigation, I never got to the point of making it a configurable option